src/main/resources/assets/mekanism/lang/ja_jp.json

```diff
@@ -0,0 +1,850 @@
+    "logistical_sorter.mekanism.auto_eject": "自動搬出:",
+    "logistical_sorter.mekanism.auto_eject.description": "接続されたインベントリに自動搬出します。",
+    "logistical_sorter.mekanism.default": "デフォルト:",
+    "logistical_sorter.mekanism.round_robin": "ラウンドロビン:",
+    "logistical_sorter.mekanism.round_robin.description": "接続された各インベントリに対して順に搬出します。",
+    "logistical_sorter.mekanism.single_item": "単一アイテム:",
+    "logistical_sorter.mekanism.single_item.description": "スタック単位ではなく単一のアイテムを搬出します(ItemStackフィルターによる最小・最大値を上書きします)。",
+    "matrix.mekanism.cells": "%s個のインダクションセル",
+    "matrix.mekanism.constituents": "構成:",
+    "matrix.mekanism.dimensions": "寸法:",
+    "matrix.mekanism.dimensions.representation": "%s x %s x %s",
+    "matrix.mekanism.induction_matrix": "インダクションマトリックス",
+    "matrix.mekanism.induction_port.configurator_mode": "インダクションポートの転送モードを%sに切り替えしました",
+    "matrix.mekanism.induction_port.output_rate": "出力レート: %s",
+    "matrix.mekanism.input_amount": "入力量:",
+    "matrix.mekanism.input_rate": "入力レート: %s/t",
+    "matrix.mekanism.output_amount": "出力量:",
+    "matrix.mekanism.output_rate": "出力レート: %s/t",
+    "matrix.mekanism.outputting_rate": "出力レート: %s/t",
+    "matrix.mekanism.providers": "%s個のインダクションプロバイダ",
+    "matrix.mekanism.receiving_rate": "受け取りレート: %s/t",
+    "matrix.mekanism.stats": "マトリックスの詳細",
+    "miner.mekanism.auto_pull": "搬入: %s",
+    "miner.mekanism.buffer_free": "空き容量: %s",
+    "miner.mekanism.config": "デジタルマイナーの設定",
+    "miner.mekanism.energy_capacity": "空きRF容量: %s",
+    "miner.mekanism.idle": "待機中",
+    "miner.mekanism.insufficient_buffer": "RF容量が足りません！",
+    "miner.mekanism.inverse": "反転モード",
+    "miner.mekanism.is_inverse": "反転: %s",
+    "miner.mekanism.low_power": "エネルギー不足です",
+    "miner.mekanism.missing_block": "ブロックがありません",
+    "miner.mekanism.paused": "一時停止",
+    "miner.mekanism.radius": "半径: %s",
+    "miner.mekanism.ready": "準備完了",
+    "miner.mekanism.require_replace": "置換するか: %s",
+    "miner.mekanism.reset": "リセット",
+    "miner.mekanism.running": "稼働中",
+    "miner.mekanism.searching": "探索中",
+    "miner.mekanism.silk_enabled": "シルク: %s",
+    "miner.mekanism.silk_touch": "シルクタッチ",
+    "miner.mekanism.to_mine": "掘削対象:",
+    "miner.mekanism.visuals": "範囲の可視化: %s",
+    "miner.mekanism.visuals.too_big": "範囲を可視化するには半径が大きすぎます",
+    "miner.mekanism.well": "順調です！",
+    "multiblock.mekanism.conflict": "衝突",
+    "multiblock.mekanism.formed": "構成済み",
+    "multiblock.mekanism.formed.chat": "マルチブロックが構成されました",
+    "multiblock.mekanism.incomplete": "未構成",
+    "network.mekanism.description": "[%s] %s個の経路、%s個の搬入出口",
+    "network.mekanism.energy": "エネルギーネットワーク",
+    "network.mekanism.fluid": "流体ネットワーク",
+    "network.mekanism.fluid_needed": "%s バケツ",
+    "network.mekanism.gas": "ガスネットワーク",
+    "network.mekanism.heat": "熱ネットワーク",
+    "network.mekanism.heat_flow": "転送先伝導 %s、環境への逸失 %s",
+    "network.mekanism.heat_flow.efficiency": "転送先伝導 %s、環境への逸失 %s、効率 %s%",
```

Comment:
The `%s%` at the end of this should just be `%s` as it was a typo initially and is what was causing #6024 to format incorrectly.